torch/_torch_docs.py

```diff
@@ -3892,6 +3892,45 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins, *, range=None, weight=None, density=False, out=None) -> (Tensor, Tensor)
+
+Computes the histogram of the values in a tensor.
+
+If :attr:`bins` is an int, it specifies the number of equal-width bins.
```

Comment:
"... bins. :attr:\`range\`..." -> "... bins, and :attr:\`range\` ..."

I think connecting these two sentences will make this a little more readable